tests/issue/109/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../../../cmake/modules/hunter_boost_component_b2_args.cmake")
+project(TestIssue109Unit)
+
+
+set(component "foo")
+set(cmakeArgs "FOO_OPT=1")
+
```

Comment:
[Boost b2 doc](http://www.boost.org/build/doc/html/bbv2/overview/invocation.html)

Seems like ```-s var=value``` can be specified many times.